server.js

```diff
@@ -66,55 +64,43 @@ app.use('/', (req, res, next) => {
   return next();
 });
 
+app.use('/*', (req, res, next) => {
+  const initialStore = Object.assign(initialState, { lastLoaded: req._parsedUrl.path });
+  global.store = configureStore(initialStore);
+  next();
+});
+
 // Init the nypl data api client.
 nyplApiClient();
 
 app.use('/*', initializePatronTokenAuth, getPatronData);
 app.use('/', apiRoutes);
 
-app.get('/*', (req, res, next) => {
-  const queryString = req._parsedUrl.query;
-  let query = {};
-  if (queryString) {
-    query = queryString
-      .split('&')
-      .map(pair => pair.split('='))
-      .reduce((acc, el) => ({ [el[0]]: el[1], ...acc }));
-  }
-
-  const location = {
-    pathname: req.originalUrl,
-    query,
-    search: '',
-  };
-
-  dataLoaderUtil.loadDataForRoutes(location, req, routeMethods, res).then(() => next());
-});
-
 app.get('/*', (req, res) => {
-  alt.bootstrap(JSON.stringify({
-    PatronStore: res.locals.data.PatronStore,
-    Store: Store.getState(),
-  },
-  ));
-
-  const appRoutes = (req.url).indexOf(appConfig.baseUrl) !== -1 ? routes().client : routes().server;
-  const title = DocumentTitle.rewind();
-  const iso = new Iso();
+  const appRoutes = (req.url).indexOf(appConfig.baseUrl) !== -1 ? routes.client : routes.server;
+  const store = global.store;
 
   match({ routes: appRoutes, location: req.url }, (error, redirectLocation, renderProps) => {
     if (error) {
       res.status(500).send(error.message);
     } else if (redirectLocation) {
       res.redirect(302, redirectLocation.pathname + redirectLocation.search);
     } else if (renderProps) {
-      application = ReactDOMServer.renderToString(<RouterContext {...renderProps} />);
-      const flushed = alt.flush();
-      iso.add(application, flushed);
+      if (!res.data) {
```

Comment:
What's this for?

Reply:
Just removed this now. It was left over from the initial approach I took, which assigned store data to `res.data` as is done in Digital Research Books [here](https://github.com/NYPL/sfr-bookfinder-front-end/blob/development/server.js#L56)